libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/ConwayFeatures.hs

```diff
@@ -301,6 +302,12 @@ vote pf govActionId =
     , otherWitsFields = []
     }
 
+assertEqualEdiff :: (Eq a, ToExpr a) => String -> a -> a -> Assertion
+assertEqualEdiff msg x y
+  | x == y = pure ()
+  | otherwise = do
+      error $ msg <> ":\n" <> show (ansiWlEditExpr $ ediff x y)
```

Comment:
We already have a function that does the same thing:
https://github.com/input-output-hk/cardano-ledger/blob/12161e72f8a718b012ec0b811f82431427f1fc94/libs/cardano-ledger-binary/testlib/Test/Cardano/Ledger/Binary/TreeDiff.hs#L140-L145